server/datastore/mysql/migrations/tables/20240116102550_AddHostActivities_test.go

```diff
@@ -0,0 +1,76 @@
+package tables
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240116102550(t *testing.T) {
+	db := applyUpToPrev(t)
+
+	// create a couple users
+	u1 := execNoErrLastID(t, db, `INSERT INTO users (name, email, password, salt) VALUES (?, ?, ?, ?)`, "u1", "u1@b.c", "1234", "salt")
+	u2 := execNoErrLastID(t, db, `INSERT INTO users (name, email, password, salt) VALUES (?, ?, ?, ?)`, "u2", "u2@b.c", "1234", "salt")
+
+	// create an activity
+	act1 := execNoErrLastID(t, db, `INSERT INTO activities (user_id, user_name, user_email, activity_type) VALUES (?, ?, ?, ?)`, u1, "u1", "u1@b.c", "act1")
+
+	// create a host execution request in the past
+	minutesAgo := time.Now().UTC().Add(-5 * time.Minute).Truncate(time.Second)
+	hsr1 := execNoErrLastID(t, db, `INSERT INTO host_script_results (host_id, execution_id, script_contents, output, created_at, updated_at) VALUES (?, ?, ?, ?, ?, ?)`, 1, uuid.NewString(), "echo 'hello'", "", minutesAgo, minutesAgo)
+
+	// Apply current migration.
+	applyNext(t, db)
+
+	// existing host execution request's timestamp hasn't changed (despite added column)
```

Comment:
❤️

Maybe it is worth adding a new helper function to encourage use in future migrations?